src/index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>New Jersey Weather Widget</title>
+  <link rel="stylesheet" href="/css/style.min.css">
+</head>
+<body>
+  <div class="container">
+    <h1 class="visually-hidden">Weather page</h1>
+    <section class="widgets">
+      <h2 class="visually-hidden">Widgets</h2>
+      <article class="widget">
+        <header class="widget__header">
+          <button type="button" class="btn widget__refresh-btn">
+            <span class="visually-hidden">refresh</span>
+          </button>
+        </header>
+        <div class="widget__body">
+          <p class="widget__degrees">79&deg;</p>
+          <div class="widget__info">
+            <p class="widget__description">Cloudy Skies</p>
+            <p class="widget__location">Sicklerville, New&nbsp;Jersey</p>
+          </div>
+          <div class="widget__date"><p>May</p><p>21</p></div>
```

Comment:
It's better to break the code like this into multiple lines for readability.

Reply:
Done 👌